client/lib/upgrades/actions/index.js

```diff
@@ -1,5 +1,13 @@
-export * from './cart';
-export * from './checkout';
-export * from './domain-management';
-export * from './domain-search';
-export * from './purchases';
+import * as cart from './cart';
+import * as checkout from './checkout';
+import * as domainManagement from './domain-management';
+import * as domainSearch from './domain-search';
+import * as purchases from './purchases';
+
+export default {
+	...cart,
+	...checkout,
+	...domainManagement,
+	...domainSearch,
+	...purchases
+};
```

Comment:
>Is this supposed to be included in this PR?

I'll look at this a bit more. It curently fails to compile with the following error with the original code:

```
ERROR in ./client/lib/upgrades/actions/index.js
Module parse failed: /Users/andrew/Documents/Code/wp-calypso/node_modules/react-hot-loader/index.js!/Users/andrew/Documents/Code/wp-calypso/node_modules/babel-loader/index.js?cacheDirectory!/Users/andrew/Documents/Code/wp-calypso/client/lib/upgrades/actions/index.js Line 5: Unexpected token
You may need an appropriate loader to handle this file type.
| 'use strict';
| 
| import _Object$defineProperty from 'babel-runtime/core-js/object/define-property';
| Object.defineProperty(exports, "__esModule", {
|   value: true
 @ ./client/my-sites/current-site/index.jsx 26:22-53
```

However, it's a bit puzzling, since the additional export supports are from a [stage 1 proposal](https://github.com/leebyron/ecmascript-more-export-from) ([source](https://github.com/tc39/ecma262)), so I'd expect them to be supported [per our configuration](https://github.com/Automattic/wp-calypso/blob/d4669869a502b040db6b83694f65019e0e7cac7f/.babelrc#L4).